gtsfm/utils/io.py

```diff
@@ -291,14 +347,28 @@ def write_images(gtsfm_data: GtsfmData, images: List[Image], save_dir: str) -> N
 
     num_imgs = gtsfm_data.number_images()
     # TODO: compute this (from keypoint data? or from track data?)
-    mean_obs_per_img = 0
+
+    image_id_num_measurements = {}
```

Comment:
It might be better to have a method in gtsfmdata to get the number of measurements for an image. @johnwlambert ?